langchain/vectorstores/rocksetdb.py

```diff
@@ -0,0 +1,308 @@
+"""Wrapper around Rockset vector database."""
+from __future__ import annotations
+
+import logging
+from enum import Enum
+from typing import Any, Iterable, List, Optional, Tuple
+
+# Make sure you have rockset client installed. If not, you can
+# install it with `pip install rockset`.
+from rockset import RocksetClient, ApiException
+from rockset.models import DeleteDocumentsRequestData
+
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.vectorstores.base import VectorStore
```

Comment:
Nit: Unless this repo does things differently, the langchain imports and rockset imports should not be separated with an empty line. It should also be alphabetic